event model - maggie - UNSURE WHERE TO PUT IM SORRY

```diff
@@ -0,0 +1,72 @@
+import { roles } from '@/types/dataModel/roles';
+import {
+  VolunteerEntity,
```

Comment:
VolunteerEntity needs to be EventEntity and EventEntity needs to be created in types/dataModel/event. It would be great if you could create that file.

also, we don't need backgroundCheckStatuses

Reply:
i wasnt sure where to put that file but it is now made